docs/en/administration/fault_diagnosis_and_analysis.md

```diff
@@ -11,15 +11,15 @@ import TabItem from '@theme/TabItem';
 
 ## Client log
 
-JuiceFS client will output logs for troubleshooting during operation, the log levels are from low to high: DEBUG, INFO, WARNING, ERROR, FATAL, by default only logs above INFO level are output. If you need to output DEBUG level logs, you need to explicitly enable it when running the JuiceFS client, such as adding the `--debug` option.
+JuiceFS client will output logs for troubleshooting during operation. Log levels in order of fatality are: DEBUG, INFO, WARNING, ERROR and FATAL. Since DEBUG logs are not printed by default, you need to explicitly enable it if you need, e.g. by adding the `--debug` option, when running the JuiceFS client, 
 
-Different JuiceFS clients obtain logs in different ways, which are described below.
+Different JuiceFS clients print logs in different ways, described as follows.
 
 ### Mount point
 
-When the JuiceFS file system is mounted with the [`-d` option](../reference/command_reference.md#juicefs-mount) (indicating running in the background), the log will be output to the system log and local log file at the same time, depending on the running user when the file system is mounted, the path of the local log file is slightly different. The log file path corresponding to root user is `/var/log/juicefs.log`, and the log file path of non-root user is `$HOME/.juicefs/juicefs.log`. For details, see [`--log` option](../reference/command_reference.md#juicefs-mount).
+When a JuiceFS file system is mounted with the [`-d` option](../reference/command_reference.md#juicefs-mount) (indicating running in the background), it will print logs to the system log file and local log file simultaneously. Depending on the user JuiceFS client running as, the path of the local log files are slightly different. The local log file locates at `/var/log/juicefs.log` when running as root, while `$HOME/.juicefs/juicefs.log` for non-root users. For details. See [`--log` option](../reference/command_reference.md#juicefs-mount).
```

Comment:
`-d` stands for `--background`, expand for clarification:

When a JuiceFS file system is mounted with the [`-d`/`--background` option](../reference/command_reference.md#juicefs-mount), it will print logs to the system log file and local log file simultaneously.

local log file path is `/var/log/juicefs.log` when running as root, `$HOME/.juicefs/juicefs.log` for non-root users.

> Shall we standardize the phrases

no need to toil just for this purpose, but better ditch the "please" in "please refer to".